kim/src/bin/kim_record_mpg

```diff
@@ -20,13 +20,15 @@
 # Author: Charles Bouveyron <charles.bouveyron@free.fr>
 #		Raphaël Pinson <raphink@raphink.net>
 
-test -n "$KDEHOME" || KDEHOME="`kde4-config --localprefix`"; export KDEHOME
+. `kf5-config --prefix`/share/kim/kim_translation
+
+#test -n "$KDEHOME" || KDEHOME="`kde4-config --localprefix`"; export KDEHOME
 #KIMDIR=`mktemp -d "$KDEHOME"/tmp-"$USER"/kim.XXXXXXXX` || exit 1
-KIMDIR=`mktemp -d "$KDEHOME"/tmp-"$HOST"/kim.XXXXXXXX` || exit 1
+KIMDIR=`mktemp -d /tmp-"$HOST"/kim.XXXXXXXX` || exit 1
```

Comment:
Well, that's because the way to get $KDEHOME  (kde4-config --localprefix) is deprecated in kf5-config (no --localprefix arguments) and I don't know of any alternatives.

All the rest are the same issue.  Any better ideas are welcome.

Reply:
I think correctly it should be `/tmp/kim.XXXXXXXX`. Many other user space tools also use `/tmp`

Reply:
[here's a SU link ](https://superuser.com/a/332616) that discusses the "standard" way of finding temp files. I guess `/tmp` should be fine for purely temporary files.

A modern alternative might be `$XDG_RUNTIME_DIR` (see [specs](https://specifications.freedesktop.org/basedir-spec/basedir-spec-latest.html) which is bound to the user session (i.e. it gets deleted when the user logs out)